### PR TITLE
Fixes RnD to Include New Weapons

### PR DESCRIPTION
**Pull request by LordFowl**
Fixes RnD to include the weapons that I added in the last Vaurca Batch.

### CODE REVIEW

**Comment by skull132** [comment]
One of these may need syndicate tech? Possibly the e-glaive, as it's a very powerful (fireaxe level) weapon without any downsides.

I also need to see what the standard levels for RnD are, and how easy it is to get the railgun. Shit's also powerful.

**Comment by LordFowl** [comment]
I'm not very familiar with RnD, so feel free to suggest balance as you please.

**Comment by OneOneThreeEight** [comment]
holy hell that's a cheap-ass railgun. without mining and just a bit of phoron from chem/xenobiology, i'm fairly certain you could produce more than 3-4 railgun prototypes to hand out to people with premining maxed research.

talk about the 2nd amendment not being infringed, huh?

anyway, if i might suggest, should add diamond and uranium requirements to it. probably silver, too.

trod packs need to be more expensive as well

**Comment by skull132** [comment]
RIP, I didn't even look at the material reqs. Yeah, no, you can't produce anything better than a spess MP-9 with the standard 3 materials. Actually, here, you can't even make that.

Silver seems to be the standard "non nuclear" weapon component. Uranium is used in the laser cannon. Minor amounts, primarily to exist as blockers.

Also, the railgun should *really* be more expensive, due to the explosion it generates.

**Comment by LordFowl** [comment]
Updated. Please review.

File: code/modules/research/designs.dm [hunk 1]
```diff
@@ -1398,39 +1398,39 @@ datum/design/item/weapon/phoronpistol
...
+	materials = list("$metal" = 8000, "$glass" = 2000, "$phoron" = 6000)
 	build_path = /obj/item/weapon/gun/energy/vaurca/blaster
 
 datum/design/item/weapon/lawgiver
 	desc = "A highly advanced firearm for the modern police force. It has multiple voice-activated firing modes."
 	id = "lawgiver"
-	req_tech = list("combat" = 6, "plasmatech" = 4, "bluespace" = 5, "materials" = 7)
+	req_tech = list("combat" = 6, "phorontech" = 4, "bluespace" = 5, "materials" = 7)
 	build_type = PROTOLATHE
 	materials = list("$metal" = 6000, "$glass" = 1000, "$uranium" = 1000, "$plasma" = 1000, "$diamond" = 3000)
```
Comment (Arrow768) [inline code/modules/research/designs.dm]: Shouldnt this be phoron ?

File: code/modules/research/designs.dm [hunk 2]
```diff
@@ -1374,6 +1374,14 @@ datum/design/item/weapon/ammo_9mm
 	materials = list("$metal" = 3750, "$silver" = 100)
 	build_path = /obj/item/ammo_magazine/c9mm
 
+datum/design/item/weapon/trod
+	id = "trod"
+	name = "tungsten rod pack"
+	desc = "moderately expensive superdense tungsten rods."
+	req_tech = list("combat" = 2, "materials" = 4)
+	materials = list("$metal" = 93750, "$gold" = 7500)
```
Comment (skull132) [inline code/modules/research/designs.dm]: Is it even possible to hold 93 750 units of metal in the RnD protolathe? Somewhere around 10 000 should be fine. For ref, 15 000 is the highest count I've been able to find.

File: code/modules/research/designs.dm [hunk 3]
```diff
@@ -1387,13 +1395,44 @@ datum/design/item/weapon/phoronpistol
 	materials = list("$metal" = 5000, "$glass" = 1000, "$phoron" = 3000)
 	build_path = /obj/item/weapon/gun/energy/toxgun
 
+datum/design/item/weapon/eglaive
+	id = "eglaive"
+	name = "energy glaive"
+	desc = "A Li'idra designed hardlight glaive reverse-engineered from schematics found amongst raider wreckages."
+	req_tech = list("combat" = 6, "phorontech" = 4, "materials" = 7, "syndicate" = 4,"powerstorage" = 4)
+	materials = list("$metal" = 10000, "$glass" = 18750, "$phoron" = 3000, "$silver" = 7500, "$diamond" = 11250)
```
Comment (skull132) [inline code/modules/research/designs.dm]: I'd say you're good to lower the amount of diamond for this. The mere presence of it on the list is a limitation enough, along with syndicate tech. With that high of a syndicate level, you could actually just remove diamond and we should be fine.

File: code/modules/research/designs.dm (hunk 2 above)
Comment (OneOneThreeEight) [inline code/modules/research/designs.dm]: Protolathe capacity expansion, probably?

**Pull request merged**